Makefile

```diff
@@ -27,6 +27,11 @@ ENVTEST_K8S_VERSION = 1.24.2
 GATEKEEPER_NAMESPACE = gatekeeper-system
 RATIFY_NAME = ratify
 
+# Local Registry Setup
+LOCAL_REGISTRY_IMAGE ?= ghcr.io/oras-project/registry:v1.0.0-rc.4
```

Comment:
We can set this to be a matrix of distributions in the build-pr workflow file. For now, it's just this default